src/Qowaiv.Validation.TestTools/Assertions/ResultAssertions.cs

```diff
@@ -1,16 +1,16 @@
-namespace FluentAssertions.Qowaiv.Validation;
+namespace Qowaiv.Validation.TestTools;
 
 /// <summary>Contains a number of methods to assert the (in)validness of a <see cref="Result"/>.</summary>
 public sealed class ResultAssertions : ResultAssertionsBase<Result>
 {
     /// <summary>Initializes a new instance of the <see cref="ResultAssertions"/> class.</summary>
-    internal ResultAssertions(Result? subject) : base(subject) { }
+    internal ResultAssertions(Result? subject, string? expression) : base(subject, expression) { }
 
     /// <summary>Asserts thats <see cref="Result"/> is invalid.</summary>
-    [CustomAssertion]
+    [Impure]
```

Comment:
This does not need the Assertion attribute?

Reply:
Yes, it does!